index.js

```diff
@@ -0,0 +1,11 @@
+const myNavv = document.getElementById('myNav');
+function openNav() {
+  myNavv.style.width = '100%';
+}
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+const openNavv = document.getElementById('.myNav');
```

Comment:
- [ ] Kindly remove scrolling from your mobile menu when it's open. That would look better.
![image](https://user-images.githubusercontent.com/91486791/223801162-f12ad379-1b00-4492-bf32-f62302675271.png)